src/App.tsx

```diff
@@ -21,6 +39,11 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={perPage}
+            onChange={(event) => {
```

Comment:
instead anonymous arrow function create function in component and pass reference here